src/app/views/templates/MrfWorkflowCompletionEmail.tsx

```diff
@@ -1,72 +1,94 @@
 }
 
 export const MrfWorkflowCompletionEmail = ({
   // Defaults are provided only for testing purposes in react-email-preview.
   formTitle = 'Test form title',
-  responseId = '64303c45828035f732088a41'
+  responseId = '64303c45828035f732088a41', 
+  formQuestionAnswers = [], 
+  outcome
 }: WorkflowEmailData): JSX.Element => {
+  const previewText = outcome
+    ?  `The outcome for ${formTitle}.` 
+    : `${formTitle} has been completed by all respondents.` 
+
   return (
     <Html>
-      <Head />
-      <Body style={outerContainerStyle}>
-        <Container style={innerContainerStyle}>
-          <Row>
-            <Column>
-              <Img src={FORMSG_LOGO_URL} alt="FormSG" />
-            </Column>
-          </Row>
-          <Row style={{ paddingTop: '32px' }}>
-            <Column>
-              <Heading style={headingStyle}>
-                {formTitle} has been completed by all respondents.
+      <Head /> 
+      <Preview>{previewText}</Preview>
+      <Body style={mainStyle}>
+          <Container style={containerStyle}>
+            <Section style={sectionStyle}>
+              <Img style={{height: '1.5rem', marginBottom: '2.5rem'}} src={FORMSG_LOGO_URL} alt="FormSG" />
+              <Heading style={headingTextStyle}>
+                {
+                  outcome 
+                    ? `The outcome for ${formTitle}` 
+                    : `${formTitle} has been completed by all respondents.` 
+                }
+              </Heading>
+              {
+                outcome ? <>
+                  <Text style={{...outcomeTextStyle, marginTop: '2.5rem', marginBottom: '0.25rem'}}>Outcome</Text>
+                  <Text style={{...outcomeTextStyle, fontWeight: 400, marginTop: '0.25rem'}}>{outcome}</Text>
+                </> : null 
+              }
+              <Hr style={{marginTop: '2.5rem', marginBottom: '2.5rem'}}/>
+              <Heading style={{...headingTextStyle, marginBottom: '2.5rem'}}>
+                Responses for {formTitle} 
               </Heading>
-            </Column>
-          </Row>
-          <Row style={{ paddingTop: '16px' }}>
-            <Column width="30%">
-              <Text style={textStyle}>Response ID</Text>
-            </Column>
-            <Column>
-              <Text style={textStyle}>{responseId}</Text>
-            </Column>
-          </Row>
-          <Row style={{ paddingTop: '32px' }}>
-            <Column>
-              <Hr />
-            </Column>
-          </Row>
-          <Row>
-            <Column>
-              <Text style={{ ...textStyle, fontSize: '14px' }}>
+              <Text style={{...titleTextStyle, marginBottom: '0.25rem'}}>Response ID</Text>
+              <Text style={{...bodyTextStyle, marginTop: '0.25rem'}}>{responseId}</Text>
+              { 
+                formQuestionAnswers.map((qa) => {
+                return <>
+                  <Text style={{...titleTextStyle, marginBottom: '0.25rem'}}>{qa.question}</Text>
```

Comment:
I'm seeing a couple of repeats of `{...titleTextStyle, marginBottom: '0.25rem'}` on texts that represent the "question" side of the submission, and `{...bodyTextStyle, marginTop: '0.25rem'}` on texts that presents the "answer" side of the submission.

Would it make sense to make these two styles as constants?

Reply:
created a `questionMargin` and `answerMargin` style, allowing us to change the padding when needed immediately. 

changed from rem to px to support gsib.